content/en/blog/_posts/2022-08-22-Auto-Refreshing-Official-CVE-Feed-is-now-in-alpha/index.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: blog
+title: "Auto-refreshing Official CVE Feed is now in Alpha"
```

Comment:
As this isn't really a Kubernetes feature, I'd play down the “alpha” part. The blog article is really to announce that there is a new feed, and the article text can make it clear that it's currently best-effort (or whatever).